src/app/src/components/ClaimedFacilitiesDetails.jsx

```diff
@@ -191,27 +189,18 @@ const InputSection = ({
             }
 
             if (isCreatable && isMultiSelect) {
-                const [
-                    optionsInExistingSet,
-                    newlyCreatedOptionsPrime,
-                ] = partition(
-                    value,
-                    v => includes(map(selectOptions, 'value'), v),
-                );
-
-                const mapStringToOption = s => ({ value: s, label: s });
-
-                return concat(
-                    map(optionsInExistingSet, mapStringToOption),
-                    map(newlyCreatedOptionsPrime, mapStringToOption),
-                );
+                return map(value, s => ({ value: s, label: s }));
```

Comment:
The deleted bit was code I wrote that in retrospect was basically useless: the point of the Createable is that any option is valid, so there's no need to intersect with the existing choices. Since we know the structure of the choices, we can just map all the values to be that.